### PR TITLE
Codify all containers swift-to-swift sync behavior

### PR DESCRIPTION
**Pull request by dbishop**
...in an integration test.

User newer container-crawler.

### CODE REVIEW

File: test/container/swift-s3-sync.conf [hunk 1]
```diff
@@ -141,13 +141,24 @@
...
             "container": "\u062acl-conn-swift-restore",
             "copy_after": 3600,
             "propagate_delete": false,
             "protocol": "swift",
             "retain_local": false,
             "restore_object": true
+        },
+        {
+            "account": "AUTH_slashacct",
+            "aws_bucket": "crazy-target",
```
Comment (timuralp) [inline test/container/swift-s3-sync.conf]: Can you change the bucket name to end with "/" or "-" so it will be easier to read the test output?

File: test/container/Dockerfile [hunk 2]
```diff
@@ -7,7 +7,7 @@ RUN apt-get update && \
 
 RUN pip install -e git://github.com/swiftstack/botocore.git@1.4.32.6#egg=botocore && \
     pip install boto3==1.3.1 && \
-    pip install -e git://github.com/swiftstack/container-crawler.git@0.0.12#egg=container-crawler && \
+    pip install -e git://github.com/swiftstack/container-crawler.git@0.0.13#egg=container-crawler && \
```
Comment (timuralp) [inline test/container/Dockerfile]: Would prefer this to be a separate commit.

File: test/container/swift-s3-sync.conf (hunk 1 above)
Comment (dbishop) [reply test/container/swift-s3-sync.conf]: Pretty sure `/` is not allowed inside a container name by Swift, but a dash could be used.  I actually did that on purpose so it'd be obvious what happens if you just stick a normal string in there.

File: test/container/Dockerfile (hunk 2 above)
Comment (dbishop) [reply test/container/Dockerfile]: Really?

**Comment by dbishop** [comment]
@timuralp updated w/suggestions
